.github/workflows/shared-ui-create-release.yml

```diff
@@ -0,0 +1,124 @@
+name: Release step 2 - create Github Release
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pages: write
+  id-token: write
```

Comment:
```suggestion
```
Can you check if any of these are required? I guess `contents: write` is needed to create the release, but what about the rest?

Reply:
you're totally correct, i've fixed that.